src/Backend/Core/Engine/Model.php

```diff
@@ -894,4 +895,9 @@ public static function updateExtraData(int $id, string $key, $value): void
         $data[$key] = $value;
         $database->update('modules_extras', ['data' => serialize($data)], 'id = ?', [$id]);
     }
+
+    public static function getEntityManager(): EntityManager
```

Comment:
Why do you need this method in Backend and Frontend Model?

Reply:
Is used in Frontend en Backend Model of Faq, is better to have a short cut, then writing it multiple times.

Reply:
Why not using `faq.repository.xxx` instead?

Reply:
Because this is the entity manager (not the repo) for persisting/removing/flushing, see https://github.com/forkcms/forkcms/pull/2362/commits/53e9c14757d4a7714304389ef55e01a2b890ae5c#diff-1f4f383328d2a945661a8580959d8de7R401 for example

Reply:
why not do those things in methods in the repository?

Reply:
Fixed ed40a9650